doc/how_to_contribute/how_to_contribute_to_site.rst

```diff
@@ -0,0 +1,73 @@
+How-To Contribute to This Site
+==============================
+
+This explains how to make and submit changes to this site.
+This how-to guide assumes you are on Ubuntu 20.04.
+
+Setting up your development environment
+---------------------------------------
+
+Follow the tutorial :doc:`/doc/tutorials/getting_started/getting_started` to set up your workspace for testing local changes.
+
+All further instructions assume you are the directory containing a local checkout of ``moveit2_tutorials``.
+
+Build and view the website locally
+----------------------------------
+
+To build the website locally run this command:
+
+.. code-block:: bash
+
+  make html
+
+Then you can use this to open the site in your webbrowser:
+
+.. code-block:: bash
+
+  open ./build/html/index.html
+
+Test for broken links
+---------------------
+
+To test for broken links run this command:
```

Comment:
Is there a time people should be doing this, or is this just a helpful command?

Reply:
Before they submit their PR they should run both this and the pre-commit change.  This is what is run in CI to test for broken links.